FandomParser/FandomParser/WikiText/WikiTextTableParser.cs

```diff
@@ -11,6 +12,9 @@ public class WikiTextTableParser
     {
         private static Dictionary<WorldRegion, Dictionary<string, string>> RegionTables { get; set; }
 
+        private static readonly Regex regexNormalizeLineEndings = new Regex(@"\r\n|\n|\r", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
This regex does not normalize strings, it matches line endings. Should probably be renamed.